### ISSUE TITLE
Render a single page

### ISSUE THREAD

**Issue by grommasdietz** [bnomei/kirby3-janitor#120]
Hi Bruno

I’m trying to create the thumbnail jobs for a current page via `janitor:render`. From my understanding it should just render the current page by default, when using it in the page blueprint like:

```yml
createJobs:
  type: janitor
  command: 'janitor:render'
  label: Create thumbnails jobs
  ```

Currently this triggers the rendering of all pages. It’s a little confusing since I expect to only render all pages when using --site flag like in the thumbs job. Is this an intended behavior? Is there a way in general to explicitly render only the current page?

(Edit: And a small side note, there seems to be an issue with the success message on buttons: `{{ count }} Seiten gerendert`)

**Comment by bnomei** [comment]
`janitor:thumbs`
used in the panel it will get --page automatically and using the --page (not the --site). thus only thumbs for current page via a panel button. even if you would call it with --site it would not work. you can currently not do that from a panel button unless you put it on the site blueprint.
if used in cli like `kirby janitor:thumbs --site` can be used to generate all thumbs.

`janitor:render` 
can be called on a page but it does not have the same logic as the thumbs command. i did not plan to keep them consistent to each other but optimize for usecases.
render has a --query which defaults to site.index and that will be used unless overwritten. so to render only the current page you would need to do `janitor:render --query page` which will be the

ℹ️ labels language have been fixed in #112

**Comment by grommasdietz** [comment]
Thanks again for explaining (and fixing)!
